dts/arm/st/h7/stm32h7_dualcore.dtsi

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2020 Moonkwun Jung
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <st/h7/stm32h7.dtsi>
+
+/ {
+	cpus {
+		#address-cells = <1>;
+		#size-cells = <0>;
+
```

Comment:
@mkainy can you check if these 2 lines are really  needed?

Reply:
![image](https://user-images.githubusercontent.com/59005828/72816503-94199480-3cab-11ea-9727-24b950fa628b.png)
The code above is stm32h7.dtsi. If I specify an address cell and a size cell in this file, do I need not to include them in stm32h7_dualcore.dtsi?

I have additional questions.
Would it be better to separate the dual core in zephyr / soc / arm / st_stm32 / stm32h7 
like dts / arm / st / h7?

Reply:
> The code above is stm32h7.dtsi. If I specify an address cell and a size cell in this file, do I need not to include them in stm32h7_dualcore.dtsi?

This should not be needed.
 
> I have additional question.
> Would it be better to separate the dual core in zephyr / soc / arm / st_stm32 / stm32h7
> like dts / arm / st / h7?

This is not stricly needed. Symbol `STM32H7_DUAL_CORE` is defined under H7 condition but is disabled by default. It is enabled as part of stm32h747i_disco board configuration. All of this remains valid with your change. We'll see later it it should be refined.